best-practices/README.md

```diff
@@ -218,7 +218,7 @@ Sass
 Browsers
 --------
 
-* Don't support versions of Internet Explorer before IE10.
+* Avoid supporting versions of Internet Explorer before IE10.
```

Comment:
This seems like a lot of extra wording to spell out how we define "avoid" in the README already.

>"Avoid" means don't do it unless you have good reason.

Does the clarity provide any benefit over what we have already?